igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildController.groovy

```diff
@@ -222,7 +222,10 @@ class BuildController {
     static void validateJobParameters(JobConfig jobConfig, Map<String, String> requestParams) {
         jobConfig.parameterDefinitionList.each { parameterDefinition ->
             String matchingParam = requestParams[parameterDefinition.name]
-            if (matchingParam != null && parameterDefinition.type == 'ChoiceParameterDefinition' && !parameterDefinition.choices.contains(matchingParam)) {
+            if (matchingParam != null &&
+                parameterDefinition.type == 'ChoiceParameterDefinition' &&
+                parameterDefinition.choices != null &&
+                !parameterDefinition.choices.contains(matchingParam)) {
```

Comment:
1. For long conditions such as this, I like to break them out into a separate (static whenever possible) method.
2. Test

Reply:
+1

Reply:
I feel like since this is already a part of a small static method i'll just add a test for that method. sound ok?

Reply:
Sounds legit, I didn't notice how small this method was already.